packages/flutter/lib/src/widgets/toggleable.dart

```diff
@@ -149,7 +150,7 @@ mixin ToggleableStateMixin<S extends StatefulWidget> on TickerProviderStateMixin
       reverseCurve: Curves.easeOut,
     );
     _reactionController = AnimationController(
-      duration: kRadialReactionDuration,
+      duration: _kRadialReactionDuration,
```

Comment:
This feels like a little bit of material leaking into widgets. Can you define a getter or a field member on widgets/ToggleableStateMixin (maybe reactionAnimationDuration?), so that in the material widgets that use it, they can override it with the material constant value?